src/strings.ts

```diff
@@ -296,9 +296,9 @@ export const strings = {
     save: "Save mapping",
     tableName: "Table name",
     title: "Map your data",
-    usbtitleImportTemplate:
+    subtitleImportTemplate:
       "Map the columns from your data source to the corresponding template fields",
-    usbtitleImportData:
+    subtitleImportData:
       "Map the columns from new data to the corresponding fields in the current chart design",
```

Comment:
It's possible this is `USB`+`title`, but Google Sheets didn't think so, and I agree w/ Google Sheets.